Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/calendar/CalendarCalculation.kt

```diff
@@ -0,0 +1,120 @@
+package de.rki.coronawarnapp.ui.calendar
+
+import org.joda.time.DateTime
+import org.joda.time.Instant
+import org.joda.time.LocalDate
+import java.util.Locale
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
```

Comment:
As this class is stateless, and not necessarily someone we want to keep in the graph for longer, mark this `@Reusable`, it will be reused, but not kept indefintely.